### ISSUE TITLE
Add GitHub Actions CI builds for easy testing

### ISSUE THREAD

**Issue by Calinou** [stuntrally/stuntrally3#6]
Similar to what https://github.com/stuntrally/stuntrally has, we should add CI builds with artifacts so you can test the latest Git version quickly. To keep artifacts small, only the Stunt Rally and sr-editor executables should be included (plus any required libraries), not the game data. You'd have to download the game data and tracks as a ZIP separately, then install the executables within the root folder of the extracted game data.

**Comment by cryham** [comment]
Right. But does #1 already add Actions? I'm not sure what the status is there.

**Comment by AnotherFoxGuy** [comment]
> But does https://github.com/stuntrally/stuntrally3/pull/1 already add Actions?

Yes, that PR adds GitHub Actions, you can a preview of it here: https://github.com/AnotherFoxGuy/stuntrally3/actions
The issue with #1 is that I've been unable to get stuntrally3 to build properly

**Comment by cryham** [comment]
Ok @Calinou but how would that ZIP with data be made?
Or shoud it be git pulled by user? Probably there should be a hash of data saying this auto build binary needs repo from this commit to work (was based on it). Data like materials, Gui layouts and others really need to be the same, not from other version, since at least Gui will crash game if it there isn't a control in .layout but is needed in code.
Anyways I'm okay if we have a 1 to 2GB zip anyway, downloads are fast, at least no data will be misplaced or so.

**Comment by cryham** [comment]
Hi @AnotherFoxGuy 
So this is being built okay since a while.
But just recently we tested Windows zip and it does not have `RenderSystem_GL3Plus.dll` which seems to be the only one working for SR3 now. Do you build it in your package ogre-next, could you add it?
As for Linux there is no zip or other pack. What needs to be done to have it auto made on builds, is it impossible? Looks the same in old SR too.

**Comment by cryham** [comment]
Yes space is an issue. I'd love to move to some hosting and not bother if we fit into 2GB limit per release on github or how is it on sourceforge IDK. But hosting is costly and I'd rather not spend time moving all, unless necessary.
Just FYI, CI builds are small here 15MB (only Windows) and only have binaries. So it's not an issue. But for big releases it could be soon.

**Comment by cryham** [comment]
Something got broken recently. Both builds are failing. @AnotherFoxGuy could you please look into it?

**Comment by AnotherFoxGuy** [comment]
@cryham The licence file got deleted: https://github.com/stuntrally/stuntrally3/commit/63441f136c9409adbe4a52d1d07682db5777c280
That file is used for the cpack installers

**Comment by cryham** [comment]
Ah thanks, I didn't realize. Changed to License.txt
